.github/workflows/pages.yml

```diff
@@ -0,0 +1,82 @@
+# Workflow for building and deploying a the /docs folder as static Jekyll site to GitHub Pages
+name: Deploy Jekyll site to Pages
+
+on:
```

Comment:
Can you add the `pull_request` event here, so the jobs listed below (minus the `deploy` one, see my other comment below) run on against PRs as well?

Reply:
Done